ui/src/landing/CreateWish.test.js

```diff
@@ -25,28 +25,34 @@ describe('CreateWish tests', () => {
     expect(wrapper.state().wishType).toEqual(wishType)
   })
 
-  describe('When name, age, and wish type are present', () => {
+  describe('When name, age, and wish type are present and valid', () => {
     let wrapper
     beforeEach(() => {
       wrapper = shallow(<CreateWish />)
       wrapper.setState({
         name: 'a name',
-        age: 'an age',
+        age: '14',
         wishType: 'a type'
       })
     })
 
     it('Should show Childinfo component', () => {
       expect(wrapper.find(Childinfo).length).toEqual(1)
     })
+
+    it('Should have validFields return true', () => {
+      const instance = wrapper.instance();
+      const {validAge} = instance.validFields(instance.state.age);
+      expect(validAge).toEqual(true);
+    })
   })
 
-  describe('When any of the states (name, age, and wish type) is absent', () => {
+  describe('When the states (name, age, and wish type) are invalid', () => {
```

Comment:
@amlidos I received a little more information on the age range for make a wish. https://wish.org/content/faq#Who-can-receive-a-wish

can you update the range to be between 2 (inclusively) and 18 (exclusively)

Reply:
Sure, I updated the code to check for that, updated the tests, and caught my fork up with the upstream master.